cli/tests/test_next_prev_commands.rs

```diff
@@ -240,7 +253,21 @@ fn test_prev_fails_on_multiple_parents() {
     // Create a merge commit, which has two parents.
     test_env.jj_cmd_ok(&repo_path, &["new", "all:@--+"]);
     test_env.jj_cmd_ok(&repo_path, &["commit", "-m", "merge"]);
-    // Advance the working copy commit.
+
+    // Check that the graph looks the way we expect.
+    insta::assert_snapshot!(get_log_output(&test_env, &repo_path), @r###"
+    @  vruxwmqvtpmx
+    ◉      yqosqzytrlsw merge
+    ├─┬─╮
+    │ │ ◉  qpvuntsmwlqt first
+    │ ◉ │  kkmpptxzrspx second
+    │ ├─╯
+    ◉ │  mzvwutvlkqwt third
+    ├─╯
+    ◉  zzzzzzzzzzzz
+    "###);
```

Comment:
perhaps, these setup could use `new root() -m` instead of `commit -m`? I was also a bit confused when adjusting the test for 3 candidates.

Reply:
Done.